src/components/Cell/helpers.js

```diff
@@ -21,6 +21,6 @@ export function getCellDefaultProps(color, size, additionalClassName) {
   return {
     style: `background-color: ${color}; width: ${size}px; height: ${size}px`,
     class: `cell block cursor-pointer rounded transition-shadow duration-100
```

Comment:
could it be a class in sass that combines all of listed? 
In general, all styles and classes related manipulations are better to move to sass layer.
Sorry, just checked, we don't have sass here...
Well, maybe postcss could handle this?

Reply:
I think tailwind can handle this!